testing/apps/details/Makefile

```diff
@@ -1,18 +1,15 @@
 APP_NAME 	:= $(shell basename `pwd`)
 IMAGE_NAME 	:= amalgam8/a8-testing-bookinfo-$(APP_NAME)
-LDFLAGS 	:=
 
-ifndef GOOS
-    GOOS := $(shell go env GOHOSTOS)
-endif
+BUILDFLAGS  := -i
+LDFLAGS 	:= -s -w -linkmode external -extldflags -static
 
-ifeq ($(GOOS),linux)
-	# linker flags to strip symbol tables and debug information
-	LDFLAGS     += -s -w
-endif
+# Integration test runs within docker containers only
+GOOS        := linux
+GOARCH      := amd64
 
 build:
-	@go build -o $(APP_NAME) -a -installsuffix nocgo -tags netgo -ldflags '$(LDFLAGS)'
```

Comment:
This build tag is not required anymore. It's part of the "old" way doing static linking. It was replaced with `-linkmode external -extldflags -static`. This is also what we now use for linking our real binaries.